src/sonic-config-engine/sonic-cfggen

```diff
@@ -345,10 +363,11 @@ def main():
         env.filters['ip_network'] = ip_network
         for attr in ['ip', 'network', 'prefixlen', 'netmask', 'broadcast']:
             env.filters[attr] = partial(prefix_attr, attr)
-        # Pass the is_multi_npu function as global
-        env.globals['multi_asic'] = is_multi_npu
```

Comment:
what about this function?

Reply:
Thanks! my local branch is as latest as a couple of weeks ago. This would explain why it is passing in my local build but not on Jenkins! You saved me time debugging it!